app/components/HeroSection.tsx

```diff
@@ -0,0 +1,18 @@
+import heroImg from "../imgs/Hero-Image.png"
```

Comment:
I recommend naming all files in `kebab-case` to match npm package naming! Any idea how we could enforce this with a GitHub action? :)